opencensus/trace/span_context.py

```diff
@@ -150,13 +150,13 @@ def check_trace_id(self, trace_id):
 
 
 def generate_span_id():
-    """Return the random generated span ID for a span.
+    """Return the random generated span ID for a span. Must be 16 digits.
 
     :rtype: int
     :returns: Identifier for the span. Must be a 64-bit integer other
               than 0 and unique within a trace.
     """
-    span_id = random.getrandbits(64)
+    span_id = random.randint(10**15, 10**16 - 1)
```

Comment:
Why did this change?

Reply:
Because the V2 API only accepts 16-digit span_id, but we can also trim the span_id in the stackdriver_exporter.

Reply:
Ok.  Maybe document that rationale, and/or make those named constants.

Reply:
Added.